sdk/storage/Azure.Storage.Common/src/Errors.cs

```diff
@@ -40,5 +53,37 @@ internal class Errors
 
         public static ArgumentException InvalidResourceType(char s) =>
             new ArgumentException($"Invalid resource type: '{s}'");
+
+        public static ArgumentException InvalidDateTimeUTC(string dateTime) =>
+            new ArgumentException($"{dateTime} must be UTC");
+
+        public static ArgumentException AccountMismatch(string accountNameCredential, string accountNameValue) =>
+            new ArgumentException(String.Format(CultureInfo.CurrentCulture, "Account Name Mismatch: {0} != {1}",
+                accountNameCredential,
+                accountNameValue));
+
+        public static ArgumentException ParseConnectionStringError() =>
```

Comment:
Nit - we don't include `Error` as a suffix in any of the other method names in this class.

Reply:
Updated. Removed 'Error' from any method containing it in the Errors class.